src/Entity/OgMembership.php

```diff
@@ -74,78 +80,75 @@ public function setCreatedTime($timestamp) {
   }
 
   /**
-   * @param mixed $etid
-   *
-   * @return OgMembership
+   * {@inheritdoc}
    */
   public function setUser($etid) {
     $this->set('uid', $etid);
     return $this;
   }
 
   /**
-   * @param mixed $fieldName
-   *
-   * @return OgMembership.
+   * {@inheritdoc}
+   */
+  public function getUser() {
+    return $this->get('uid')->value;
+  }
```

Comment:
This is an entity reference field, so the value will return the user entity object, no? Likewise, setting a value for an ER field will also work with an ID or an entity object? Unless you mean something else completely? :)